drive_node:
  ros__parameters:
    command_model: 'differential_drive'
    auto_max_speed_characterization: False
    auto_cmd_model_characterization: False
    wheel_radius: 0.165
    wheel_baseline: 0.55
    cmd_rate: 20
    encoder_rate: 30
    max_lin_speed: 1.0
    max_ang_speed: 1.0
    step_len: 6.0
    n_calib_steps: 3
    dead_man_button: True
    dead_man_index: 5
    dead_man_threshold: 0.7
    calib_trigger_button: False
    calib_trigger_index: 7
    calib_threshold: 0.7
